scripts/hotlink/changes.mjs

```diff
@@ -0,0 +1,35 @@
+export const changes = [
+  // react
+  {
+    path: "/packages/react/package.json",
+    entry: "./src/index.ts",
+    exports: {
+      ".": "./src/index.ts",
+      "./evm": "./src/evm/index.ts",
+      "./solana": "./src/solana/index.ts",
+      "./evm/connectors/magic": "./src/evm/connectors/magic.ts",
+      "./evm/connectors/gnosis-safe": "./src/evm/connectors/gnosis-safe.ts",
+    },
+  },
+  // react-core
+  {
+    path: "/packages/react-core/package.json",
+    entry: "./src/index.ts",
+    exports: {
+      ".": "./src/index.ts",
+      "./evm": "./src/evm/index.ts",
+      "./solana": "./src/solana/index.ts",
+    },
+  },
+  // sdk
+  // {
+  //   path: "/packages/sdk/package.json",
+  //   entry: "./src/index.ts",
+  //   exports: {
+  //     ".": "./src/index.ts",
+  //     "./evm": "./src/evm/index.ts",
+  //     "./solana": "./src/solana/index.ts",
+  //     "./solana/server": "./src/solana/server/index.ts",
+  //   },
+  // },
```

Comment:
so the problem with sdk is that extra import that breaks things?

Reply:
@joaquim-verges yeah pretty much. I'm also not sure if react, react-core is working properly. Did not get a chance to check - had to move to another task. 

But will probably make it nicer today.